### ISSUE TITLE
False Positive for `Lint/ImplicitStringConcatenation` with prism

### ISSUE THREAD

**Issue by pjsk-stripe** [rubocop/rubocop#12878]
## Expected behavior

`Lint/ImplicitStringConcatenation` behaves strangely with prism in the situation below. With prism turned off, it doesn't report an error.

I imagine this is some combination of the `\n` and heredocs. But strangely the prefix `Something:` is determining the output.

## Actual behavior
```
$ rubocop --debug
For /Users/pjsk/Workspace/rubocop-test: configuration from /Users/pjsk/Workspace/rubocop-test/.rubocop.yml
Default configuration from /Users/pjsk/.rbenv/versions/3.3.0/lib/ruby/gems/3.3.0/gems/rubocop-1.63.4/config/default.yml
Use parallel by default.
Running parallel inspection
Loading cache from /Users/pjsk/.cache/rubocop_cache/534273b9ca3f55074640a50c5ca18229f9ff297e/6d7a3b621ca1730e04accd938619e4bdab66cfb1/63a887baec0f59303e2489405d3b0e5e5e5d0d31
Inspecting 2 files
Scanning /Users/pjsk/Workspace/rubocop-test/Gemfile
Loading cache from /Users/pjsk/.cache/rubocop_cache/534273b9ca3f55074640a50c5ca18229f9ff297e/6d7a3b621ca1730e04accd938619e4bdab66cfb1/b34fb59736dff2e5d1f73f75d34d9efc96a08880
.Scanning /Users/pjsk/Workspace/rubocop-test/test.rb
Loading cache from /Users/pjsk/.cache/rubocop_cache/534273b9ca3f55074640a50c5ca18229f9ff297e/6d7a3b621ca1730e04accd938619e4bdab66cfb1/63a887baec0f59303e2489405d3b0e5e5e5d0d31
W

Offenses:

test.rb:5:1: W: Lint/ImplicitStringConcatenation: Combine  and   Something:  into a single string literal, rather than using implicit string concatenation.
  Something: #{b}
^^^^^^^^^^^^^

2 files inspected, 1 offense detected
Finished in 0.2889069998636842 seconds
```

## Steps to reproduce the problem

```
$ rubocop -V
1.63.4 (using Prism 0.27.0, rubocop-ast 1.31.3, running on ruby 2.7.2) [arm64-darwin22]
```

```yaml
AllCops:
  TargetRubyVersion: 3.3
  ParserEngine: parser_prism
  NewCops: enable
```

```rb
# frozen_string_literal: true

_note = <<~SOMEHEREDOC
  Thing: #{a} \n
  Something: #{b}
SOMEHEREDOC
```

## RuboCop version

Include the output of `rubocop -V` or `bundle exec rubocop -V` if using Bundler.
If you see extension cop versions (e.g. `rubocop-performance`, `rubocop-rspec`, and others)
output by `rubocop -V`, include them as well. Here's an example:

```
$ rubocop -V
1.63.4 (using Prism 0.27.0, rubocop-ast 1.31.3, running on ruby 3.3.0) [arm64-darwin23]
```

**Comment by koic** [comment]
This issue is a dup with #12834.

**Comment by pjsk-stripe** [comment]
Hi @koic 

it's possible this is a dup of https://github.com/rubocop/rubocop/issues/12834, but that issue is resolved on prism 0.30.0. The issue above reproduces today on 0.30.0.

```
$ rubocop --debug
For /Users/pjsk/Workspace/rubocop-test: configuration from /Users/pjsk/Workspace/rubocop-test/.rubocop.yml
Default configuration from /Users/pjsk/.rbenv/versions/3.3.0/lib/ruby/gems/3.3.0/gems/rubocop-1.64.1/config/default.yml
Use parallel by default.
Running parallel inspection
Loading cache from /Users/pjsk/.cache/rubocop_cache/b4a4c5eda0898195be6c14a78d1d3bdfb3bc2652/6d7a3b621ca1730e04accd938619e4bdab66cfb1/f0957a62676848f54e4cbcc894aac4afe1e36e60
Loading cache from /Users/pjsk/.cache/rubocop_cache/b4a4c5eda0898195be6c14a78d1d3bdfb3bc2652/6d7a3b621ca1730e04accd938619e4bdab66cfb1/6ba38c5e0c3edaa9051125abcf77c14aaba1e855
Inspecting 2 files
Scanning /Users/pjsk/Workspace/rubocop-test/Gemfile
Loading cache from /Users/pjsk/.cache/rubocop_cache/b4a4c5eda0898195be6c14a78d1d3bdfb3bc2652/6d7a3b621ca1730e04accd938619e4bdab66cfb1/6ba38c5e0c3edaa9051125abcf77c14aaba1e855
.Scanning /Users/pjsk/Workspace/rubocop-test/test.rb
Loading cache from /Users/pjsk/.cache/rubocop_cache/b4a4c5eda0898195be6c14a78d1d3bdfb3bc2652/6d7a3b621ca1730e04accd938619e4bdab66cfb1/f0957a62676848f54e4cbcc894aac4afe1e36e60
W

Offenses:

test.rb:5:1: W: Lint/ImplicitStringConcatenation: Combine  and   Something:  into a single string literal, rather than using implicit string concatenation.
  Something: #{b}
^^^^^^^^^^^^^

2 files inspected, 1 offense detected
Finished in 0.2301639998331666 seconds
```

```
rubocop -V
1.64.1 (using Prism 0.30.0, rubocop-ast 1.31.3, running on ruby 3.3.0) [arm64-darwin23]
```

```
$ cat .rubocop.yml
AllCops:
  TargetRubyVersion: 3.3
  ParserEngine: parser_prism
  NewCops: enable%
```

```
$ cat test.rb
# frozen_string_literal: true

_note = <<~SOMEHEREDOC
  Thing: #{a} \n
  Something: #{b}
SOMEHEREDOC
```

**Comment by viralpraxis** [comment]
The issue still persists:

```ruby
<<~TXT
  Lorem Lorem Lorem:\r\n  \r\n
TXT
```

```
W: [Correctable] Lint/ImplicitStringConcatenation: Combine  and  into a single string literal, rather than using implicit string concatenation.
```

rubocop -V: `1.66.1 (using Parser 3.3.5.0, rubocop-ast 1.32.3, running on ruby 3.3.5) [x86_64-linux]`, `prism 1.0.0`

**Comment by KieranP** [comment]
Getting the same issue on latest version of Rubocop and Prism:

```ruby
        abort <<~MSG
          \n\n\e[31mDeployment is already in progress
          Started #{time_diff} ago by #{deploy_user}
          Run 'bin/cap <stage> lock:release' if previous deploy failed/terminated\e[0m\n
        MSG
```

Whats worse, is autocorrect gets stuck in a loop:

```
lib/capistrano/tasks/lock.rake:12:1: W: [Corrected] Lint/ImplicitStringConcatenation: Combine  and           Started  into a single string literal, rather than using implicit string concatenation. Or, if they were intended to be separate method arguments, separate them with a comma.
          Started #{time_diff} ago by #{deploy_user}
^^^^^^^^^^^^^^^^^^
lib/capistrano/tasks/lock.rake:12:1: W: [Corrected] Lint/ImplicitStringConcatenation: Combine  and  +           Started  into a single string literal, rather than using implicit string concatenation. Or, if they were intended to be separate method arguments, separate them with a comma.
 +           Started #{time_diff} ago by #{deploy_user}
^^^^^^^^^^^^^^^^^^^^^
lib/capistrano/tasks/lock.rake:12:1: W: [Corrected] Lint/ImplicitStringConcatenation: Combine  and  +  +           Started  into a single string literal, rather than using implicit string concatenation. Or, if they were intended to be separate method arguments, separate them with a comma.
 +  +           Started #{time_diff} ago by #{deploy_user}
^^^^^^^^^^^^^^^^^^^^^^^^
lib/capistrano/tasks/lock.rake:12:1: W: [Corrected] Lint/ImplicitStringConcatenation: Combine  and  +  +  +           Started  into a single string literal, rather than using implicit string concatenation. Or, if they were intended to be separate method arguments, separate them with a comma.
 +  +  +           Started #{time_diff} ago by #{deploy_user}
^^^^^^^^^^^^^^^^^^^^^^^^^^^
lib/capistrano/tasks/lock.rake:12:1: W: [Corrected] Lint/ImplicitStringConcatenation: Combine  and  +  +  +  +           Started  into a single string literal, rather than using implicit string concatenation. Or, if they were intended to be separate method arguments, separate them with a comma.
 +  +  +  +           Started #{time_diff} ago by #{deploy_user}
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
lib/capistrano/tasks/lock.rake:12:1: W: [Corrected] Lint/ImplicitStringConcatenation: Combine  and  +  +  +  +  +           Started  into a single string literal, rather than using implicit string concatenation. Or, if they were intended to be separate method arguments, separate them with a comma.
 +  +  +  +  +           Started #{time_diff} ago by #{deploy_user}
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
lib/capistrano/tasks/lock.rake:12:1: W: [Corrected] Lint/ImplicitStringConcatenation: Combine  and  +  +  +  +  +  +           Started  into a single string literal, rather than using implicit string concatenation. Or, if they were intended to be separate method arguments, separate them with a comma.
 +  +  +  +  +  +           Started #{time_diff} ago by #{deploy_user}
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
lib/capistrano/tasks/lock.rake:12:1: W: [Corrected] Lint/ImplicitStringConcatenation: Combine  and  +  +  +  +  +  +  +           Started  into a single string literal, rather than using implicit string concatenation. Or, if they were intended to be separate method arguments, separate them with a comma.
 +  +  +  +  +  +  +           Started #{time_diff} ago by #{deploy_user}
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
lib/capistrano/tasks/lock.rake:12:1: W: [Corrected] Lint/ImplicitStringConcatenation: Combine  and  +  +  +  +  +  +  +  +           Started  into a single string literal, rather than using implicit string concatenation. Or, if they were intended to be separate method arguments, separate them with a comma.
 +  +  +  +  +  +  +  +           Started #{time_diff} ago by #{deploy_user}
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
lib/capistrano/tasks/lock.rake:12:1: W: [Corrected] Lint/ImplicitStringConcatenation: Combine  and  +  +  +  +  +  +  +  +  +           Started  into a single string literal, rather than using implicit string concatenation. Or, if they were intended to be separate method arguments, separate them with a comma.
 +  +  +  +  +  +  +  +  +           Started #{time_diff} ago by #{deploy_user}
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
lib/capistrano/tasks/lock.rake:12:1: W: [Corrected] Lint/ImplicitStringConcatenation: Combine  and  +  +  +  +  +  +  +  +  +  +           Started  into a single string literal, rather than using implicit string concatenation. Or, if they were intended to be separate method arguments, separate them with a comma.
 +  +  +  +  +  +  +  +  +  +           Started #{time_diff} ago by #{deploy_user}
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
lib/capistrano/tasks/lock.rake:12:1: W: [Corrected] Lint/ImplicitStringConcatenation: Combine  and  +  +  +  +  +  +  +  +  +  +  +           Started  into a single string literal, rather than using implicit string concatenation. Or, if they were intended to be separate method arguments, separate them with a comma.
 +  +  +  +  +  +  +  +  +  +  +           Started #{time_diff} ago by #{deploy_user}
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
lib/capistrano/tasks/lock.rake:12:1: W: [Corrected] Lint/ImplicitStringConcatenation: Combine  and  +  +  +  +  +  +  +  +  +  +  +  +           Started  into a single string literal, rather than using implicit string concatenation. Or, if they were intended to be separate method arguments, separate them with a comma.
 +  +  +  +  +  +  +  +  +  +  +  +           Started #{time_diff} ago by #{deploy_user}
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
lib/capistrano/tasks/lock.rake:12:1: W: [Corrected] Lint/ImplicitStringConcatenation: Combine  and  +  +  +  +  +  +  +  +  +  +  +  +  +           Started  into a single string literal, rather than using implicit string concatenation. Or, if they were intended to be separate method arguments, separate them with a comma.
 +  +  +  +  +  +  +  +  +  +  +  +  +           Started #{time_diff} ago by #{deploy_user}
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
lib/capistrano/tasks/lock.rake:12:1: W: [Corrected] Lint/ImplicitStringConcatenation: Combine  and  +  +  +  +  +  +  +  +  +  +  +  +  +  +           Started  into a single string literal, rather than using implicit string concatenation. Or, if they were intended to be separate method arguments, separate them with a comma.
 +  +  +  +  +  +  +  +  +  +  +  +  +  +           Started #{time_diff} ago by #{deploy_user}
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
lib/capistrano/tasks/lock.rake:12:1: W: [Corrected] Lint/ImplicitStringConcatenation: Combine  and  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +           Started  into a single string literal, rather than using implicit string concatenation. Or, if they were intended to be separate method arguments, separate them with a comma.
 +  +  +  +  +  +  +  +  +  +  +  +  +  +  +           Started #{time_diff} ago by #{deploy_user}
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
lib/capistrano/tasks/lock.rake:12:1: W: [Corrected] Lint/ImplicitStringConcatenation: Combine  and  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +           Started  into a single string literal, rather than using implicit string concatenation. Or, if they were intended to be separate method arguments, separate them with a comma.
 +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +           Started #{time_diff} ago by #{deploy_user}
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
lib/capistrano/tasks/lock.rake:12:1: W: [Corrected] Lint/ImplicitStringConcatenation: Combine  and  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +           Started  into a single string literal, rather than using implicit string concatenation. Or, if they were intended to be separate method arguments, separate them with a comma.
 +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +           Started #{time_diff} ago by #{deploy_user}
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
lib/capistrano/tasks/lock.rake:12:1: W: [Corrected] Lint/ImplicitStringConcatenation: Combine  and  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +           Started  into a single string literal, rather than using implicit string concatenation. Or, if they were intended to be separate method arguments, separate them with a comma.
 +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +           Started #{time_diff} ago by #{deploy_user}
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
lib/capistrano/tasks/lock.rake:12:1: W: [Corrected] Lint/ImplicitStringConcatenation: Combine  and  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +           Started  into a single string literal, rather than using implicit string concatenation. Or, if they were intended to be separate method arguments, separate them with a comma.
 +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +           Started #{time_diff} ago by #{deploy_user}
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
lib/capistrano/tasks/lock.rake:12:1: W: [Corrected] Lint/ImplicitStringConcatenation: Combine  and  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +           Started  into a single string literal, rather than using implicit string concatenation. Or, if they were intended to be separate method arguments, separate them with a comma.
 +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +           Started #{time_diff} ago by #{deploy_user}
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
lib/capistrano/tasks/lock.rake:12:1: W: [Corrected] Lint/ImplicitStringConcatenation: Combine  and  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +           Started  into a single string literal, rather than using implicit string concatenation. Or, if they were intended to be separate method arguments, separate them with a comma.
 +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +           Started #{time_diff} ago by #{deploy_user}
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
lib/capistrano/tasks/lock.rake:12:1: W: [Corrected] Lint/ImplicitStringConcatenation: Combine  and  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +           Started  into a single string literal, rather than using implicit string concatenation. Or, if they were intended to be separate method arguments, separate them with a comma.
 +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +           Started #{time_diff} ago by #{deploy_user}
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
lib/capistrano/tasks/lock.rake:12:1: W: [Corrected] Lint/ImplicitStringConcatenation: Combine  and  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +           Started  into a single string literal, rather than using implicit string concatenation. Or, if they were intended to be separate method arguments, separate them with a comma.
 +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +           Started #{time_diff} ago by #{deploy_user}
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
lib/capistrano/tasks/lock.rake:12:1: W: [Corrected] Lint/ImplicitStringConcatenation: Combine  and  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +           Started  into a single string literal, rather than using implicit string concatenation. Or, if they were intended to be separate method arguments, separate them with a comma.
 +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +           Started #{time_diff} ago by #{deploy_user}
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
lib/capistrano/tasks/lock.rake:12:1: W: [Corrected] Lint/ImplicitStringConcatenation: Combine  and  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +           Started  into a single string literal, rather than using implicit string concatenation. Or, if they were intended to be separate method arguments, separate them with a comma.
 +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +           Started #{time_diff} ago by #{deploy_user}
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
lib/capistrano/tasks/lock.rake:12:1: W: [Corrected] Lint/ImplicitStringConcatenation: Combine  and  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +           Started  into a single string literal, rather than using implicit string concatenation. Or, if they were intended to be separate method arguments, separate them with a comma.
 +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +           Started #{time_diff} ago by #{deploy_user}
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
lib/capistrano/tasks/lock.rake:12:1: W: [Corrected] Lint/ImplicitStringConcatenation: Combine  and  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +           Started  into a single string literal, rather than using implicit string concatenation. Or, if they were intended to be separate method arguments, separate them with a comma.
 +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +           Started #{time_diff} ago by #{deploy_user}
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
lib/capistrano/tasks/lock.rake:12:1: W: [Corrected] Lint/ImplicitStringConcatenation: Combine  and  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +           Started  into a single string literal, rather than using implicit string concatenation. Or, if they were intended to be separate method arguments, separate them with a comma.
 +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +           Started #{time_diff} ago by #{deploy_user}
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
lib/capistrano/tasks/lock.rake:12:1: W: [Corrected] Lint/ImplicitStringConcatenation: Combine  and  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +           Started  into a single string literal, rather than using implicit string concatenation. Or, if they were intended to be separate method arguments, separate them with a comma.
 +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +           Started #{time_diff} ago by #{deploy_user}
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
lib/capistrano/tasks/lock.rake:12:1: W: [Corrected] Lint/ImplicitStringConcatenation: Combine  and  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +           Started  into a single string literal, rather than using implicit string concatenation. Or, if they were intended to be separate method arguments, separate them with a comma.
 +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +           Started #{time_diff} ago by #{deploy_user}
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
lib/capistrano/tasks/lock.rake:12:1: W: [Corrected] Lint/ImplicitStringConcatenation: Combine  and  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +           Started  into a single string literal, rather than using implicit string concatenation. Or, if they were intended to be separate method arguments, separate them with a comma.
 +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +           Started #{time_diff} ago by #{deploy_user}
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
lib/capistrano/tasks/lock.rake:12:1: W: [Corrected] Lint/ImplicitStringConcatenation: Combine  and  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +           Started  into a single string literal, rather than using implicit string concatenation. Or, if they were intended to be separate method arguments, separate them with a comma.
 +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +           Started #{time_diff} ago by #{deploy_user}
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
lib/capistrano/tasks/lock.rake:12:1: W: [Corrected] Lint/ImplicitStringConcatenation: Combine  and  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +           Started  into a single string literal, rather than using implicit string concatenation. Or, if they were intended to be separate method arguments, separate them with a comma.
 +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +           Started #{time_diff} ago by #{deploy_user}
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
lib/capistrano/tasks/lock.rake:12:1: W: [Corrected] Lint/ImplicitStringConcatenation: Combine  and  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +           Started  into a single string literal, rather than using implicit string concatenation. Or, if they were intended to be separate method arguments, separate them with a comma.
 +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +           Started #{time_diff} ago by #{deploy_user}
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
lib/capistrano/tasks/lock.rake:12:1: W: [Corrected] Lint/ImplicitStringConcatenation: Combine  and  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +           Started  into a single string literal, rather than using implicit string concatenation. Or, if they were intended to be separate method arguments, separate them with a comma.
 +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +           Started #{time_diff} ago by #{deploy_user}
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
lib/capistrano/tasks/lock.rake:12:1: W: [Corrected] Lint/ImplicitStringConcatenation: Combine  and  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +           Started  into a single string literal, rather than using implicit string concatenation. Or, if they were intended to be separate method arguments, separate them with a comma.
 +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +           Started #{time_diff} ago by #{deploy_user}
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
lib/capistrano/tasks/lock.rake:12:1: W: [Corrected] Lint/ImplicitStringConcatenation: Combine  and  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +           Started  into a single string literal, rather than using implicit string concatenation. Or, if they were intended to be separate method arguments, separate them with a comma.
 +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +           Started #{time_diff} ago by #{deploy_user}
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
lib/capistrano/tasks/lock.rake:12:1: W: [Corrected] Lint/ImplicitStringConcatenation: Combine  and  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +           Started  into a single string literal, rather than using implicit string concatenation. Or, if they were intended to be separate method arguments, separate them with a comma.
 +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +           Started #{time_diff} ago by #{deploy_user}
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
lib/capistrano/tasks/lock.rake:12:1: W: [Corrected] Lint/ImplicitStringConcatenation: Combine  and  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +           Started  into a single string literal, rather than using implicit string concatenation. Or, if they were intended to be separate method arguments, separate them with a comma.
 +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +           Started #{time_diff} ago by #{deploy_user}
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
lib/capistrano/tasks/lock.rake:12:1: W: [Corrected] Lint/ImplicitStringConcatenation: Combine  and  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +           Started  into a single string literal, rather than using implicit string concatenation. Or, if they were intended to be separate method arguments, separate them with a comma.
 +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +           Started #{time_diff} ago by #{deploy_user}
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
lib/capistrano/tasks/lock.rake:12:1: W: [Corrected] Lint/ImplicitStringConcatenation: Combine  and  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +           Started  into a single string literal, rather than using implicit string concatenation. Or, if they were intended to be separate method arguments, separate them with a comma.
 +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +           Started #{time_diff} ago by #{deploy_user}
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
lib/capistrano/tasks/lock.rake:12:1: W: [Corrected] Lint/ImplicitStringConcatenation: Combine  and  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +           Started  into a single string literal, rather than using implicit string concatenation. Or, if they were intended to be separate method arguments, separate them with a comma.
 +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +  +           Started #{time_diff} ago by #{deploy_user}
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

**Comment by Earlopain** [comment]
Will be fixed in the next prism release https://github.com/ruby/prism/pull/3347

**Comment by pjsk-stripe** [comment]
@Earlopain thanks so much! Looks like it solves things on my end 🙏

**Comment by dvandersluis** [comment]
Prism 1.4.0 has been released which should fix this issue. Please upgrade the gem and try again.
